collections/open-journalism/index.md

```diff
@@ -0,0 +1,18 @@
+---
+items:
+ - fivethirtyeight/data
+ - NYTimes/objective-c-style-guide
+ - newsapps/beeswithmachineguns
+ - voxmedia/meme
+ - propublica/guides
+ - censusreporter/censusreporter
```

Comment:
@brandonrosage just trying to pad his line count with all these separate item lines, smh. 😛